deeppavlov/dataset_readers/morphotagging_dataset_reader.py

```diff
@@ -22,35 +22,40 @@
 from deeppavlov.core.data.utils import download_decompress, mark_done
 
 WORD_COLUMN, POS_COLUMN, TAG_COLUMN = 1, 3, 5
+HEAD_COLUMN, DEP_COLUMN = 6, 7
 
 log = getLogger(__name__)
 
-
 def get_language(filepath: str) -> str:
     """Extracts language from typical UD filename
     """
     return filepath.split("-")[0]
 
-
 def read_infile(infile: Union[Path, str], from_words=False,
```

Comment:
```suggestion
def read_infile(infile: Union[Path, str], *, from_words=False,
```
So that changing arguments order would not affect anything anymore

Reply:
I have never seen such syntax before.

Reply:
https://www.python.org/dev/peps/pep-3102/